setup/www/host_vars/.gitignore

```diff
@@ -1 +1,2 @@
-infra-*
\ No newline at end of file
+infra-*
+!node-www.tmpl
```

Comment:
what's this for?

Reply:
We're tracking this file https://github.com/nodejs/build/blob/master/setup/www/host_vars/node-www.tmpl
(keeping it ignored, make my IDE angry)